frontend/ui/widget/imagewidget.lua

```diff
@@ -355,17 +356,31 @@ function ImageWidget:paintTo(bb, x, y)
         h = size.h
     }
     logger.dbg("blitFrom", x, y, self._offset_x, self._offset_y, size.w, size.h)
+    -- Figure out if we're trying to render one of our own icons...
+    local not_an_icon = (not self.file or (self.file and not util.stringStartsWith(self.file, "resources/")))
```

Comment:
Isn't `local is_icon = self.file and util.stringStartsWith(self.file, "resources/")))` just more readable, and below `if ... and not is_icon then` ?